packages/calypso-e2e/src/lib/pages/gutenberg-editor-page.ts

```diff
@@ -289,16 +294,9 @@ export class GutenbergEditorPage {
 	 * @param {boolean} visit Whether to then visit the page.
 	 * @returns {Promise<void} No return value.
 	 */
-	async publish( {
-		visit = false,
-		saveDraft = false,
-	}: { visit?: boolean; saveDraft?: boolean } = {} ): Promise< string > {
+	async publish( { visit = false }: { visit?: boolean } = {} ): Promise< string > {
```

Comment:
The intention here is to reduce reliance on using one method and spamming it with more and more parameters, correct?

Reply:
Actually, the intermediate `saveDraft` step is not needed anymore when we target both `View Post` buttons - if one disappears (post-publish panel), the other is always available (bottom-left popup message). I thought it would be right to just remove it then since it was added only because of the disappearing post-panel issue. Does that make sense?

Reply:
> it was added only because of the disappearing post-panel issue. Does that make sense?
Yep, it makes sense and wanted to be clear that I'm totally okay with this change. 
I felt that one of the issues with the Selenium framework was the eventual parameter spamming that made it difficult to tweak those methods. I've been updating the docs on GitHub and am going to include a section on preferring smaller methods that do one or two things only, to codify this approach.